main/views.py

```diff
@@ -13,3 +14,8 @@ def index(request):
         request,
         "index.html",
     )
+
+
+def cms_signin_redirect_to_site_signin(request):
+    """Redirect wagtail admin signin to site signin page"""
+    return redirect_to_login(reverse("wagtailadmin_home"), login_url="/signin")
```

Comment:
`reverse()` isn't imported so this is throwing an error, it could probably use a test too.